src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    `Hello, world!`
```

Comment:
нет необходимости в кавычках
```suggestion
    Hello, world!
```

Reply:
Ошибку исправил

Reply:
Я не могу понять, Hello world  нужно писать в кавычках или без них? Изначально написал без кавычек. На проверке сказали что Кавычки нужны )) что делать ?  Задание не проходит проверку